common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp

```diff
@@ -731,6 +735,49 @@ geometry_msgs::msg::Pose calcInterpolatedPose(
 
   return output_pose;
 }
+
+inline geometry_msgs::msg::Vector3 createVector3(const double x, double y, double z)
+{
+  return geometry_msgs::build<geometry_msgs::msg::Vector3>().x(x).y(y).z(z);
+}
+
+inline geometry_msgs::msg::Twist createTwist(
+  const geometry_msgs::msg::Vector3 & velocity, geometry_msgs::msg::Vector3 & angular)
+{
+  geometry_msgs::msg::Twist twist;
+  twist.linear = velocity;
+  twist.angular = angular;
+  return twist;
```

Comment:
[nits] I believe you can use `geometry_msgs::build` also here. It enables us to ensure all fields of the message are initialized.

Reply:
fixed at [7ab88dc](https://github.com/autowarefoundation/autoware.universe/pull/2187/commits/7ab88dc7f3f73b3ed158a8110caabb72891f9e31)